metrics.yaml

```diff
@@ -218,3 +218,17 @@ setting_event:
         description: The location that this setting was changed from
         type: string
     expires: never
+telemetry_event:
+  telemetry_id:
```

Comment:
I haven't done a ton of metrics stuff, but if I'm reading this right, it's just one new event being added to broadcast the install id via Glean? The install id should be an extra field sent along with each event type in metrics.yaml (setting_event, launch_event and startup_event). I'm not even sure we need an event for it if we are sending it with every other metrics event. Did a discussion happen I missed and we only want to send this with a new event? If so, someone should update this doc again to indicate the changes: https://docs.google.com/document/d/1jTxG4kyDsgwAVHYkDvHvABmO85-wk9vppJe6ikNe76A/edit

If any changes to how often the install id has to be fetched cause issues with performance (since that will be calling into another application and that application will be reading from disk each time) maybe it can be cached in a global javascript variable like so?

```
let installIdCachedInMemoryInsteadOfASetting = undefined;

function getInstallId() {
  if (installIdCachedInMemoryInsteadOfASetting !== undefined) {
    return installIdCachedInMemoryInsteadOfASetting;
  }

  // .. get install id from Native messaging host
  installIdCachedInMemoryInsteadOfASetting = retrievedInstallId; // keep it in memory until the extension is unloaded
  return installIdCachedInMemoryInsteadOfASetting;
}
```